frontend/src/react/Copy/CopyList/CopyList.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react';
+import moment from 'moment';
+import {connect} from 'react-redux';
+import {getExamsForStudent} from '../../../redux/exams/actions/get';
+import CopyListDisplayer from "./CopyListDisplayer";
+import {sortExamsBySessionDate} from "../../../helpers/exam";
+
+class CopyList extends Component {
+    //TODO Mapper uniquement les exams dont l'élève fait partie du groupe: exemple, l'élève est dans EBM, on lui mappe ses exams uniquement pas ceux des autres groupes.
+     state = {
+        exams: {},
+    };
+
+    async componentDidMount() {
+        await this.props.fetchExams();
+        if(this.props.exams) {
```

Comment:
Can you comment a bit more here ? 
The code looks super complex and there's a line that's really long 🙈